IEXSharp/Helper/ExecutorREST.cs

```diff
@@ -68,18 +68,15 @@ public ExecutorREST(HttpClient client, string publishableToken, string secretTok
 					content = await responseContent.Content.ReadAsStringAsync();
 					if (responseContent.StatusCode == HttpStatusCode.OK)
 					{ // Successful Request
-						if (!typeof(IEnumerable).IsAssignableFrom(typeof(ReturnType))
-							&& content == "[]")
-						{ // if not expecting an array but receive an empty array, return null
-							return null;
+						if (typeof(IEnumerable).IsAssignableFrom(typeof(ReturnType))
+							&& content[0] != '[')
+						{ // if expecting an array but receive a single item instead, create a new List with that single item
+							content = '[' + content + ']';
```

Comment:
I feel like this is a step in the right direction and better than just returning null.

Reply:
The next question is, if we do want to return an error. And if we do, what error to return? Do we create our own error since IEX did not provide one? But then the user would get an error that's not on the list in the IEX doc, and would wonder where it came from. Or we could use one of the IEX errors. But that might not be good either bc IEX did not send that error, but the user might think they did.